src/filename_inspector.py

```diff
@@ -17,8 +18,19 @@
 INPUT_FAIL_ON_VIOLATION = "INPUT_FAIL_ON_VIOLATION"
 
 
-def get_action_input(name: str) -> str:
-    return os.getenv(name)
+def get_action_list_input(name: str, lower: bool = True) -> list[str]:
```

Comment:
```suggestion
def get_action_list_input(name: str) -> list[str]:
```
`lower` does not seem to be used

Reply:
Removed in commit TBD.